packages/components/app/styles/components/code-block.scss

```diff
@@ -0,0 +1,355 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CODE-BLOCK
+//
+
+.hds-code-block-wrapper {
+  position: relative;
+}
+
+.hds-code-block__copy-button {
+  position: absolute;
+  right: 16px;
+  bottom: 16px;
+}
+
+.hds-code-block--has-line-wrapping {
+  white-space: pre-wrap;
+}
```

Comment:
This would need to go after the `.hds-code-block` definition, otherwise it gets overridden and `@lineWrapping={{true}}` has no effect